libraries/extensions/ros2-bridge/python/src/typed/mod.rs

```diff
@@ -65,13 +66,14 @@ mod tests {
             let path = PathBuf::from(env!("CARGO_MANIFEST_DIR")); //.join("test_utils.py"); // Adjust this path as needed
 
             // Add the Python module's directory to sys.path
-            py.run_bound(
-                "import sys; sys.path.append(str(path))",
+            py.run(
+                &CString::new("import sys; sys.path.append(str(path))")
+                    .expect("Unable to build CString"),
```

Comment:
```suggestion
            py.run(
                c_str!("import sys; sys.path.append(str(path))")
```

This for example should not be failing so let's use the macro indicated as within the documentation.
https://pyo3.rs/main/doc/pyo3/marker/struct.python#method.run

Reply:
Can you make this change for all CString?